docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -0,0 +1,300 @@
+|===
+
+[discrete]
+[[fleet-server-monitoring]]
+== {fleet-server} monitoring
+
+//Suggestion: Talk about the metrics and logs that {fleet-server} provides, how
+//users can enable and see them, how to use them to determine when to scale up
+//{fleet-server}.
+
+Monitoring {fleet-server} is key since the operation of the {fleet-server} is
+paramount to the health of the deployed agents and the services they offer. When
+{fleet-server} is not operating correctly, it may lead to delayed check-ins,
+status information, and updates for the agents it manages. The monitoring data
+will tell you when to add capacity for {fleet-server}, and provide error logs
+and information to troubleshoot other issues.
+
+To enable monitoring for {fleet-server}, turn on agent monitoring in the agent
+policy. For self-managed clusters, monitoring is on by default when you create a
+new agent policy or use the existing Default {fleet-server} agent policy.
+However, it is off by default in the {ecloud} agent policy because monitoring
+requires additional RAM.
+
+To turn on {fleet-server} monitoring in the agent policy:
+
+. In {fleet}, go to *Agent Policies* and click on the *{ecloud} agent policy*.
++
+[role="screenshot"]
+image::images/fleet-policy-page.png[Fleet Policy Page]
+
+. Click the *Settings* tab and notice that Agent monitoring is
+off by default.
+
+. Under *Agent monitoring*, select *Collect agent logs* and
+*Collect agent metrics*.
++
+--
+[role="screenshot"]
+image::images/elastic-cloud-agent-policy-page.png[{ecloud} Policy Page]
+
+The agent will now be able to collect logs and metrics from the {fleet-server}.
+
+NOTE: The {fleet-server} is deployed as yet another agent in the system.
+--
+
+. Next, set the *Default namespace*.
++
+Setting the default namespace lets you segregate {fleet-server} monitoring data
+from other collected data. This makes it easier to search and visualize the
+monitoring data. By default the monitoring data is sent to the *default*
+namespace.
+
+. To confirm your change, click *Save changes*.
+
+To see the metrics collected for {fleet-server}, go to *Analytics > Discover*.
+
+In the following example, `fleetserver` was configured as the namespace, and
+you can see the metrics collected:
+
+[role="screenshot"]
+image::images/dashboard-with-namespace-showing.png[Namespace]
+
+[role="screenshot"]
+image::images/datastream-namespace.png[Datastream]
+
+In {kib}, go to *Analytics > Dashboard* and search for the predefined dashboard
+called *[Elastic Agent] Agent metrics*. Choose this dashboard, and run a query
+based on the `fleetserver` namespace.
+
+The following dashboard shows data for the query `data_stream.namespace:
+"fleetserver"`. In this example, you can observe CPU and memory usage as a
+metric and then resize the {fleet-server}, if necessary.
+
+[role="screenshot"]
+image::images/dashboard-datastream.png[Dashboard Datastream]
+
+// Nima: These steps work, but I wonder how this is different from what you see
```

Comment:
Should be the same. just wanted to show how the user could look at monitoring data for the fleet server

Reply:
I think showing the query might be useful if folks want to filter further, so I'll keep it, but also mention that you can deselect other metrics to view fleet_server only.